operator/v1/types.go

```diff
@@ -42,24 +42,29 @@ var (
 // inside of the Spec struct for your particular operator.
 type OperatorSpec struct {
 	// managementState indicates whether and how the operator should manage the component
+	// +optional
 	ManagementState ManagementState `json:"managementState"`
```

Comment:
this we might want to require. Wdyt?

Reply:
@damemi for inspiration: here we would want to define an enum and automatically generate an OpenAPI regex out of it, something like `// +openapi:enum=Managed,Unmanaged`.

Reply:
:+1: